ops/testing.py

```diff
@@ -866,7 +951,7 @@ def set_model_info(self, name: str = None, uuid: str = None) -> None:
         self.set_model_name(name)
         self.set_model_uuid(uuid)
 
-    def set_model_name(self, name: str) -> None:
+    def set_model_name(self, name: Optional[str]) -> None:
```

Comment:
Seems like `name` should not be optional here.  Looks like the culprit is set_model_info.  We should probably modify the impl of that method to only forward calls to set_model_name, and set_model_uuid if the associated kwarg is not None.

Reply:
Seems like an innocent changes, if we trust that nobody was using set_model_info to set the model name back to None...

Reply:
@rwcarlsen I implemented the change as you suggested. If you and @pengale are +1 that this is an acceptable risk of breaking someone's code, we can go with it and close this conversation.